infra/gcp/main.tf

```diff
@@ -209,20 +275,24 @@ resource "google_compute_global_address" "google_managed_services_default" {
 
 resource "google_service_networking_connection" "private_vpc_connection" {
   network = google_compute_network.default.id
-  service = "servicenetworking.googleapis.com"
-  reserved_peering_ranges = [google_compute_global_address.google_managed_services_default.name]
+  service = "services/servicenetworking.googleapis.com"
+  reserved_peering_ranges = [
+    "jg-test-clone-resource-id-ip-range",
```

Comment:
Can we make a note somewhere to remove this when Jackie no longer needs this. Or some lifecycle ignore where there can be more ranges added by hand?

Reply:
`jg-test-clone-resource-id-ip-range` is currently not in used by anything.

It appears that db-jcaq4 was created in some special (perhaps outdated?) way. It doesn't have a range associated. 

It seems to me that the right fix is this:
1. Delete `jg-test-clone-resource-id-ip-range`.
2. Add `cloud-sql-ip-range`
3. Make `cloud-sql-ip-range` the reserved peering range for this VPC peering.

Do you agree? If yeah, I can talk to Jackie Monday and make sure we don't interrupt her workflow.